lib/rewrite/rewrite-expr-grammar.ym

```diff
@@ -103,6 +103,7 @@ rewrite_expr
         | KW_SET '(' rewrite_template_content
           {
             last_rewrite = log_rewrite_set_new($3, configuration);
+            last_template_options = log_rewrite_set_get_value_template_options(last_rewrite);
             log_template_unref($3);
           }
```

Comment:
I don't see the way to pass the `LogTemplateOptions` pointer trough `log_matcher_replace()`:

```
log_matcher_replace(self->matcher, *pmsg, self->super.value_handle, value, length, self->replacement, &new_length);
```